README.md

```diff
@@ -68,63 +69,100 @@ contract="contract SimpleStorage =
-
-{"arguments":[{"type":"word","value":42}],"function":"set"}
 ```
 
-And second using the Sophia contract source which returns the Sophia types and values:
+Returns:
+`{"arguments":[{"type":"word","value":42}],"function":"set"}`
+
+#### With Sophia Source
+
+And secondly, using the Sophia contract source which returns the Sophia types and values:
 
 ```
 curl -H "Content-Type: application/json" -d "{\"calldata\":\"cb_AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACA6hZQte0c6B/XQTuHZwWpc6rFreRzqkolhGkTD+eW6BwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACoA4Uun\",\"function\":\"set\",\"source\":\"$contract\"}" -X POST http://localhost:3080/decode-calldata/source
+```
 
+Returns:
+```
 {"arguments":[{"type":[105,110,116],"value":[52,50]}],"function":"set"}
```

Comment:
Side note: You really want to have the strings as list of integers?!

Reply:
I left this endpoint to Nikita and Robert to sort out what the output format should be, but list of integers wasn't what I expected :-)

Reply:
thanks @hanssv , didn't catch my attention - slipped some Erlang in there @rvirding ? ;)

Reply:
I asked Hans directly and left them as they were and waited for his response.

Reply:
To be on the safe side, yes. This forces everyone to cast to suitable number types instead of just taking everything for an integer right away.